src/gtc/passes/gtir_pipeline.py

```diff
@@ -38,7 +38,12 @@ def __init__(self, node: gtir.Stencil):
         self._cache: Dict[Tuple[PASS_T, ...], gtir.Stencil] = {}
 
     def steps(self) -> Sequence[PASS_T]:
-        return [check_assignments, prune_unused_parameters, resolve_dtype, upcast]
+        return [
+            check_assignments,
+            prune_unused_parameters,
+            resolve_dtype,
+            upcast,
```

Comment:
Remove comma here and format to remove the changed line.

Reply:
TIL that black honors those commas.